__tests__/test-data.test.js

```diff
@@ -50,3 +49,54 @@ describe("api/topics", () => {
     );
   });
 });
+
+describe("GET /api/articles", () => {
+  test("returns article api ", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("200,check if it returns the right keys ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const arrayOfArticles = response.body.data;
+
+        expect(arrayOfArticles[0]).toHaveProperty("title");
+        expect(arrayOfArticles[0]).toHaveProperty("topic");
+        expect(arrayOfArticles[0]).toHaveProperty("author");
+        expect(arrayOfArticles[0]).toHaveProperty("body");
+      });
+  });
+  test("200,returns the correct obj ", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then((response) => {
+        const articleObj = response.body.data;
+        const articleId = articleObj.article_id;
+        expect(articleId).toBe(2);
+        expect(articleObj.title).toBe(articleData[1].title);
+        expect(articleObj.topic).toBe(articleData[1].topic);
+        expect(articleObj.author).toBe(articleData[1].author);
+        expect(articleObj.body).toBe(articleData[1].body);
+        //        expect(arrayOfArticles[0]).toBe(articleData[2])
+      });
+  });
+
+  test("404, no column found with htis specific name  ", () => {
```

Comment:
You're not checking columns, you're checking if an article exists in the database. The test inside this description is fine, just watch how you're describing your tests.